package/yast2-storage-ng.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Nov 16 10:31:04 UTC 2017 - jlopez@suse.com
+
+- Improve logging: add wrapper param for guard method and a xml
+  representation of the devicegraph in dumped into logs.
```

Comment:
"IS dumped" I guess :-). But, since you have to touch it, I think is easier to parse without the passive: "add wrapper param for guard method and dump xml representation of the devicegraph to the logs.